src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,173 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    static class Node<T> {
+
+        private T value;
+        private Node<T> prev;
+        private Node<T> next;
+
+        Node(T data) {
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor. In this case, it should accept `T value`, `Node<T> prev`, and `Node<T> next`.

Reply:
ok, done